### PR TITLE
Added javadoc comments to FlutterActivity and FlutterFragmentActivity.

### PR DESCRIPTION
**Pull request by matthew-carroll**
Added javadoc comments to FlutterActivity and FlutterFragmentActivity.

### CODE REVIEW

File: shell/platform/android/io/flutter/embedding/android/FlutterActivity.java [hunk 1]
```diff
@@ -187,6 +187,10 @@
  *     android:value="true"
  *     />
  * }
+ * <p>
+ * A number of methods in this class have the same implementation as
```
Comment (xster) [inline shell/platform/android/io/flutter/embedding/android/FlutterActivity.java]: Should these be javadocs? I don't think people want to see it in https://api.flutter.dev/javadoc/ right? Maybe split this into 2 parts. Reference the other alternative in javadocs for users and leave a code comment for other maintainers.

**Comment by xster** [comment]
LGTM

File: shell/platform/android/io/flutter/embedding/android/FlutterActivity.java (hunk 1 above)
Comment (matthew-carroll) [reply shell/platform/android/io/flutter/embedding/android/FlutterActivity.java]: Sounds good. Made the changes.

Comment (xster) [reply shell/platform/android/io/flutter/embedding/android/FlutterActivity.java]: LG

**Pull request merged**